api/src/formatters/csv.ts

```diff
@@ -29,9 +29,32 @@ export const csvParser: Parser = async (data: string) => {
   };
 };
 
+const csvInjectionProtector = (str: string) => {
+  const riskyChars = ["=", "+", "-", "@", ",", ";", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+
+  if(!str) return "";
+
+  const firstChar = str.charAt(0);
+  const isInjected = riskyChars.includes(firstChar);
+  if(!isInjected) return str;
+
+  const sliceStr = str.slice(1);
+  return csvInjectionProtector(sliceStr);
+}
+
 export const csvExporter: Exporter = async (data: IntermediateTranslationFormat) => {
+  
+  // clear some characters
+  const protectedTerm = csvInjectionProtector(data.translations[0].term);
```

Comment:
@bvenceslas can you explain the logic here? Why we are using only the first element of the "translations" array? What will happen with other elements? Or it's always just one element!? Don't you think we cut the processing of the array to a single element with your code changes?

Reply:
oh Yes!
let me rework on it

Reply:
Yes please rework and see if all working both in UI and also run Unit Tests (there was 1 or 2 tests failing before, so run tests BEFORe and AFTER you changes)